shared/params/config.go

```diff
@@ -165,7 +165,7 @@ var defaultBeaconConfig = &BeaconChainConfig{
 	DepositsForChainStart: 16384,
 	RandBytes:             3,
 	SyncPollingInterval:   6 * 1, // Query nodes over the network every slot for sync status.
-	BatchBlockLimit:       100,
+	BatchBlockLimit:       50,
```

Comment:
why changed it to 50?

Reply:
This was just a param I was testing with to get sync to work. We can change it back